helpers/weather_scraper.rb

```diff
@@ -0,0 +1,75 @@
+require 'mechanize'
+require 'pry'
+require_relative 'weather_day'
+
+
+class WeatherScraper
+  attr_reader :forecast
+  def initialize(location)
+    @location = location
+    @weatherman = Mechanize.new
+    @page = get_page
+    @calendar = get_calendar
+    @next_ten_days = assemble_next_ten_days
+    @forecast = get_forecast
+  end
+
+  def render
+    html = ""
+    @forecast.each do |day|
+      html += "<tr>"
+      html += "<td>#{day.date}</td>"
+      html += "<td>#{day.high_temp}</td>"
+      html += "<td>#{day.low_temp}</td>"
+      html += "<td>#{day.classification}</td>"
+      html += "</tr>"
+    end
+    return html
+  end
+
+  private
+
+  attr_reader :location, :weatherman, :page, :calendar, :next_ten_days
+
+  def get_page
+    wunderground_url = "http://www.wunderground.com/cgi-bin/findweather/getForecast?query=" + location.to_s
+    weatherman.get(wunderground_url)
```

Comment:
veeeeeeeeeeeeeeeeeeeeeery clever. I see a lot of rubyists using the approach of string modification to go where they want; I need to up my game in that arena.